### PR TITLE
No, Shenglong

### PR DESCRIPTION
**Pull request by nickbarnwell**

### CODE REVIEW

**Comment by zeteg** [comment]
Bans should really be extracted into a separate module, and not hard-coded in.

**Comment by nickbarnwell** [comment]
[pg disagrees](http://paulgraham.com/ds.html).

**Comment by zeteg** [comment]
http://www.courts.ca.gov/documents/nc100.pdf

**Comment by nickbarnwell** [comment]
http://www.courts.ca.gov/1054.htm#acc3164

File: app/services/get_or_create_user.rb [hunk 1]
```diff
@@ -13,9 +13,11 @@ def self.call(params, home_city)
       val[:new_user?] = false
     else
       password = Devise.friendly_token.first(8)
-      user = User.create(params.merge({home_city: home_city,
-                                       password: password}).symbolize_keys)
-      UserMailer.delay.registration(user, password)
+      unless (/Shenglong Gao/i).match(params[:name])
```
Comment (rcchen) [inline app/services/get_or_create_user.rb]: Seriously @nickbarnwell?

Comment (nickbarnwell) [inline app/services/get_or_create_user.rb]: :shipit:

**Pull request closed without merging**